docs/guides/setup/setup_overview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Setup: Overview"
+---
+# [![Setup Icon](../../images/universal_map/Gear-active.png)](setup_overview.md) Setup: Overview 
+
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='active' connect='inactive' create='inactive' validate='inactive'/>
+
+<!-- Only keep one of the 'To best understand this document' lines.  For processes like the Universal Map steps, use the first one.  For processes like the Architecture Reviews, use the second one. -->
+
+:::note Prerequisites
+- To best understand this document, you must have completed Step 1: Setup of the [Getting Started tutorial](../../tutorials/getting_started/initialize_a_data_context.md).
+:::
+	
+Getting started with Great Expectations is quick and easy.  Once you have completed your setup, you will have access to all of the features of Great Expectations from a single entry point: Your Data Context.  You will also have your Metadata Stores and Data Docs configured in the manner most suitable for your project's purposes.
+
+## The Setup process
+
+<!-- Brief outline of what the process entails.  -->
+
+Setup entails ensuring your system is prepared to run Great Expectations, installing Great Expectations itself, initializing your deployment, and doing some optional configuration of things like Data Docs and Datastores.  We'll look at each of these things in sequence.
+
+<!-- The following subsections should be repeated as necessary.  They should give a high level map of the things that need to be done or optionally can be done in this process, preferably in the order that they should be addressed (assuming there is one). If the process crosses multiple steps of the Universal Map, use the <SetupHeader> <ConnectHeader> <CreateHeader> and <ValidateHeader> tags to indicate which Universal Map step the subsections fall under. -->
+
+### 1. System Dependencies
+
+The first thing to take care of is making sure your work environment has the utilities you need to install and run Great Expectations.  These include a working Python install (version 3.8 or greater), the ability to pip install Python packages, an internet connection, and a browser so that you can use Jupyter notebooks.  Best practices are to use a virtual environment for your project's workspace.
```

Comment:
Is version 3.8 our minimum? If it's not already, this should be publicly documented.

Reply:
I'm not certain if this is our minimum, but it is the version our on boarding guidance tells us to install.

Reply:
Understood. Don and/or James should be able to give you an official version.

Reply:
I've updated this according to feedback from Don.